src/server/command/_list.zig

```diff
@@ -3,6 +3,7 @@ pub const gamemode = @import("gamemode.zig");
 pub const help = @import("help.zig");
 pub const invite = @import("invite.zig");
 pub const kill = @import("kill.zig");
+pub const setSpawn = @import("setSpawn.zig");
```

Comment:
Please keep this sorted alphabetically.
Also please call it just `spawn` it can also serve as a way to query the spawn position.